### PR TITLE
use dep

### PR DESCRIPTION
**Pull request by xh3b4sd**

### CODE REVIEW

File: Gopkg.toml [hunk 1]
```diff
@@ -20,6 +20,10 @@
 #  version = "2.4.0"
 
 
+[[constraint]]
+  name = "github.com/aws/aws-sdk-go"
+  version = "1.12.3"
+
```
Comment (xh3b4sd) [inline Gopkg.toml]: This diff doesn't make any sense. The file is completely new.

**Comment by xh3b4sd** [comment]
One job in circle is on hold. Is there a reason for this? Should we wait? How long does it take usually?

**Comment by rossf7** [comment]
If you approve the job then the e2e tests will be run. That is optional but I like to run it on larger changes. It is run anyway when you merge to master.

The tests take about 8 minutes. As they launch a guest cluster, check its health and that scaling works.

**Comment by rossf7** [comment]
You can approve the job using the web UI or just merge it. Up to you :)

**Pull request merged**